core-api/src/main/java/com/optimizely/ab/event/internal/EventFactory.java

```diff
@@ -168,6 +168,11 @@ public LogEvent createConversionEvent(@Nonnull ProjectConfig projectConfig,
         List<Attribute> attributesList = new ArrayList<Attribute>();
 
         for (Map.Entry<String, ?> entry : attributes.entrySet()) {
+            //Ignore null value attribute
+            if (entry.getValue() == null) {
```

Comment:
Can we whitelist strings, numbers (probably just `int`s and `double`s for now, but the team might need to discuss more), and booleans rather than blacklisting some of the values we don't want to send? I should mention that OASIS-3537 has been updated and might not have mentioned that initially. 😬

Reply:
@nchilada this is related to OASIS-3535 not for OASIS-3537

Reply:
Oh, interesting. @mikeng13 can you clarify the difference between 3535 and 3537?

Reply:
As per my understanding, this is only related to NULL values, while OASIS-3537 is more related to filtering out all unsupported values but we have hold-off for OASIS-3537.

Reply:
I've deprecated 3537 in favor of 3535. So the requirement from @nchilada for this particular PR is right!